### PR TITLE
Adding terror.dino.icu

### PR DESCRIPTION
**Pull request by 4H4CayJ**
## Description

this will be my padfolio website i am slowly building it for now i have a poem written on it i plan to add my socials and show case my projects in future

### CODE REVIEW

**Comment by jaspermayone** [comment]
Hey @4H4CayJ, can you tell me a bit more about what this is?

**Comment by 4H4CayJ** [comment]
> Hey @4H4CayJ, can you tell me a bit more about what this is?

this will be my padfolio website  i am slowly building it for now i have a poem written on it i plan to add my socials and show case my projects in future 

repo is private i have set the domains want me to revert to show it ?

**Comment by jaspermayone** [comment]
Hey @4H4CayJ,

At Hack Club we want to do our best to be as respectful as possible! Please make a new PR with a different / more appropriate subdomain.

File: dino.icu.yaml [hunk 1]
```diff
@@ -426,3 +426,8 @@ node:
 - ttl: 600
   type: CNAME
   value: tgh75972.github.io.
+
+gyat:
```
Comment (jaspermayone) [inline dino.icu.yaml:430]: Hey @4H4CayJ,

At Hack Club we want to do our best to be as respectful as possible! Please edit your PR with a different / more appropriate subdomain.

**Review by jaspermayone — CHANGES_REQUESTED** [review]

File: dino.icu.yaml (hunk 1 above)
Comment (4H4CayJ) [reply dino.icu.yaml]: ok i changed it to terror .

File: dino.icu.yaml [hunk 2]
```diff
@@ -426,3 +426,8 @@ node:
 - ttl: 600
   type: CNAME
   value: tgh75972.github.io.
+
+terror: 
+  ttl: 600 
+  type: CNAME 
+  value: 4H4CayJ.github.io
```
Comment (jaspermayone) [inline dino.icu.yaml:433]: please add a trailing dot here.

```yaml
value: 4H4CayJ.github.io.
```

**Review by jaspermayone — CHANGES_REQUESTED** [review]

**Review by jaspermayone — APPROVED** [review]
LGTM! 🎉

**Pull request merged**